cypress/tsconfig.json

```diff
@@ -1,12 +1,9 @@
 {
+  "extends": "../tsconfig.json",
   "compilerOptions": {
     "strict": true,
     "baseUrl": "../node_modules",
-    "target": "es5",
-    "lib": [
-      "es5",
-      "dom"
-    ],
+    "isolatedModules": false,
```

Comment:
The base tsconfig sets this to `true`, but Cypress uses namespaces, so this cannot be `true`